std_msgs/Bool start
---
nav_msgs/Path grasps
